modules/fileout_netcdf/FONcTransform.cc

```diff
@@ -1390,6 +1451,54 @@ void FONcTransform::check_and_obtain_dimensions_internal(D4Group *grp) {
     }
 
 }
+void FONcTransform::set_constraint_var_dio_flag(libdap::Array* t_a) const {
```

Comment:
It might be the odd way GitHub formats the diffs, but please take a look at the formatting of this  function and adjust if needed.